plugins/experiment_procedure.cc

```diff
@@ -407,7 +407,7 @@ run(const ProcedureRunConfig & run,
         if(runProcConf.outputAccuracyDataset) {
             PolyConfigT<Dataset> outputPC;
             outputPC.id = ML::format("%s_results_%d", runProcConf.experimentName, (int)progress);
-            outputPC.type = "sparse.mutable";
```

Comment:
it's possible to specify a default type while still keeping it optional. @guyd can you point to an example of this plz?